app/bundles/EmailBundle/Entity/EmailRepository.php

```diff
@@ -137,15 +136,32 @@ public function getEntities($args = array())
             if (!isset($args['email_type']) || $args['email_type'] == 'list') {
                 $q->leftJoin('e.lists', 'l');
             }
+            $q->leftJoin('e.stats', 'es');
+            $q->addSelect('MAX(es.dateSent) AS max_date_sent');
+            $q->addGroupBy('e.id');
         }
 
         $args['qb'] = $q;
 
-        return parent::getEntities($args);
+        $result = parent::getEntities($args);
+
+        // @todo Replace with resultSetMapping or something.
```

Comment:
I've tried with resultSetMapping at first - but it turned out that `addFieldResult` works only for columns from the same table, it refuses to work with "virtual" column (which is "MAX(es.dateSent)"). I guess there should be better way than just `foreach`ing whole array (maybe custom hydrator) - but as is for now.